src/core/composables/learnersCreation.js

```diff
@@ -111,10 +111,11 @@ export const useLearnersCreation = (
     };
```

Comment:
C'est pas vraiment lié a ta PR car c'etait deja ecrit comme ca dans le code mais je trouve etrange et je pense que ce n'est pas un bonne pratique de passer la computed `companies` en argument de la composable plutot que de passer la valeur de la computed. Est-ce que tu sais pourquoi on l'a ecrit comme ca ?

Reply:
pour garder la réactivité, mais sinon je peux la passer en réf pour ne pas passer la computed en props ce qui peut en effet être bizarre mais passre une référence de la computed https://stackoverflow.com/questions/70855278/how-to-pass-reactive-data-to-a-composable-in-vue-3